README.md

```diff
@@ -854,6 +855,33 @@ datetime
 
 ```
 
+
+
+### Top Holders Percent Of Total Supply
+
+Returns the top holders' percent of total supply - in exchanges, outside exchanges and combined.
+
+[Premium metric](#premium-metrics)
+
+```python
+san.get(
+    "top_holders_percent_of_total_supply/ethereum",
```

Comment:
@tspenov @kalkov how should we proceed here where we have time restrictions? For `slug` based metics we can show `sentiment`s free data but here we do not have this option

Reply:
We can add to the premium metrics section 
```markdown
```
something like: